tools/setup-windows/setup-windows.csproj

```diff
@@ -1,38 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{73DF9E10-E933-4222-B8E1-F4536FFF9FAD}</ProjectGuid>
+    <TargetFrameworks>net472;netcoreapp3.1</TargetFrameworks>
+    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
     <OutputType>Exe</OutputType>
     <RootNamespace>setupwindows</RootNamespace>
     <AssemblyName>setup-windows</AssemblyName>
+    <OutputPath>$(XamarinAndroidSourcePath)bin\$(Configuration)\bin\</OutputPath>
+    <DefaultItemExcludes>$(DefaultItemExcludes);**\.DS_Store</DefaultItemExcludes>
```

Comment:
Do we need this in every project? Can we put it in a new `Directory.Build.props` or `Configuration.props`?

Reply:
> Do we need this in every project?

I would add it for sure, so IDE is not polluted with random files.

How? I don't know. `Directory.Build.props` or `Configuration.props`? I'm OK with either. I don't know the rules of engagement (yet). (can I mess around with those or not... etc)

Reply:
I asked the MSBuild team about this. Short-form MSBuild projects look like they should already ignore anything with a `.` prefix:

https://github.com/dotnet/sdk/blob/f04f823a3fdf38e6739c906164737dc93016b9ee/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.targets#L37-L40

I think you can just remove this? Can you share a screenshot of what you see in the IDE?